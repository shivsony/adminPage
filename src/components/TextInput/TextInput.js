import { withStyles, InputBase } from '@material-ui/core';

const BootstrapInput = withStyles((theme) => ({
  root: {
    'label + &': {
      marginTop: theme.spacing(3),
    },
  },
  input: {
    borderRadius: 4,
    position: 'relative',
    backgroundColor: theme.palette.background.paper,
    border: `1px solid ${theme.palette.primary.main}`,
    fontSize: 16,
    padding: '5px',
    transition: theme.transitions.create(['border-color', 'box-shadow']),
  },
}))(InputBase);

const TextInput = BootstrapInput;
export default TextInput;