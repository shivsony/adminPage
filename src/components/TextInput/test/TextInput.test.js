
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';


import TextInput from '../TextInput';



test('render without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<TextInput id="textinput" />, div);
});

test('matchs snapshot', () => {
    const tree = renderer.create(<TextInput id="textinput" />).toJSON();
    expect(tree).toMatchSnapshot();
});
