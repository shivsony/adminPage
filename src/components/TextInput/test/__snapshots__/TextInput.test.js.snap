// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matchs snapshot 1`] = `
<div
  className="MuiInputBase-root WithStyles(ForwardRef(InputBase))-root-1"
  onClick={[Function]}
>
  <input
    className="MuiInputBase-input WithStyles(ForwardRef(InputBase))-input-2"
    id="textinput"
    onAnimationStart={[Function]}
    onBlur={[Function]}
    onChange={[Function]}
    onFocus={[Function]}
    type="text"
  />
</div>
`;
